pyccl/__init__.py

```diff
@@ -16,6 +16,12 @@
 # SWIG-generated
 from . import ccllib as lib
 
+# monkey patch for isitgr and fast-pt if Numpy>=1.24
+from packaging.version import parse
+import numpy as np
```

Comment:
And one more. Apologies for doing it incrementally! Can we make sure we remove `np` and `parse` from pyccl's namespace after the check?